shared/domain/src/main/java/gov/va/vro/model/event/EventReason.java

```diff
@@ -0,0 +1,47 @@
+package gov.va.vro.model.event;
+
+import lombok.Getter;
+
+/**
+ * The object lists events that can happen in VRO process. Each event has a code and a narrative
+ * message. It can be used to handle the event message consistently in the VRO process.
+ *
+ * @author warren @Date 4/6/23
+ */
+@Getter
+public enum EventReason {
+  SUFFICIENCY_UNDETERMINED("assessorError", "Sufficiency cannot be determined."),
+  NEW_NOT_PRESUMPTIVE(
+      "newClaimMissingFlash266", "New claim cannot be determined to be presumptive."),
+  PDF_UPLOAD_FAILED_AFTER_ORDER_EXAM(
+      "docUploadFailed", "PDF upload failed after exam order requested."),
+  PDF_UPLOAD_FAILED_AFTER_RFD("docUploadFailedRfd", "Failed to upload PDF file."),
+  EXAM_ORDER_FAILED("examOrderFailed", "Failed to order exam."),
+  ANNOTATIONS_FAILED("annotationDataRequestFailed", "Failed to get annotation data."),
+  BIP_UPDATE_FAILED("bipUpdateFailed", "BIP update failed.");
+
+  private static int MAX_CODE_LENGTH = 50;
+
+  private String code;
```

Comment:
This PR is a work in progress. The original Jira ticket only mentioned one message. The discussions in the PR led to this solution.

As the magic strings (codes) were used in the code for messages, I guess, @au2 suggested to add a narrative and kept the code. It would avoid too much impact on the existing code while we make the event message handling a bit more consistent.

Since the code values are extracted from the current codes, I add MAX_CODE_LENGTH to avoid long string search.

If we know all the events, we can redefine the enum here and just use a simple enum without code and narrative later. At present, using code and narrative will reduce the possibility to break other parts in the existing code.